cpu/cortexm_common/include/bit.h

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_cortexm_common
+ * @{
+ *
+ * @file
+ * @brief       Bit access macros for Cortex-M based CPUs
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ */
+
+#ifndef BITBAND_H
```

Comment:
Hm, somehow this file triggers an encoding bug in the headercheck script. I suspect the beautiful a with a circle on top in your name. ;)
Unfortunately I cannot reproduce the bug locally.

Anyhow, this define (and the closing endif comment) need to be adapted to the file rename.
```dist/tools/headerguards/check.sh | patch -p0``` should do that for you.